usagov/usagov-2021
# USAGOV-1103: Check what are these IPs for in Firewall rules.

Researched the origins and purposes of a handful of firewall rules that have come into question. See the ticket for more information.


https://cm-jira.usa.gov/browse/USAGOV-1103

## Description
Commented out two rules that I believe are no longer needed. Let me know if I should delete them instead.
Added comments outlining how two other rules are AWS EC2 instances on the west-coast.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions
After deploying to Dev, we need to make sure nothing is crashing. We should confirm any side services, like Cron, etc. are still working.

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
